src/components/toolbox/buttons/css/baseV2.css

```diff
@@ -1,56 +1,53 @@
 @import '../../../app/variablesV2.css';
+@import '../../../app/mixins.css';
 
 @define-mixin buttonBasics {
+  @mixin contentNormal bold;
+
   width: 250px;
   max-width: 100%;
   border-radius: var(--border-radius-standard);
-  box-shadow: var(--box-shadow-button);
   text-transform: none;
-  font-weight: var(--font-weight-bold);
-  font-family: var(--content-font) !important;
-  font-size: var(--button-font-size);
+  outline: none;
   transition: var(--transition-button) !important;
-}
 
-@define-mixin buttonPrimaryV2 {
-  background-color: var(--color-primary-standard) !important;
-  color: var(--color-white) !important;
+  & .ripple {
+    display: none;
+  }
 
   &:disabled {
-    opacity: 0.8;
+    opacity: 0.58;
   }
+}
+
+@define-mixin buttonPrimaryV2 {
+  background-color: var(--color-ultramarine-blue) !important;
+  color: var(--color-white) !important;
 
   &:focus,
   &:hover {
-    background: #004aff !important;
+    background: color(var(--color-ultramarine-blue) + l(+4%)) !important;
+    box-shadow: 3px 3px 10px 0 rgba(61, 113, 255, 0.36) !important;
```

Comment:
this can be a constant

Reply:
Not necessary right now, this box-shadow is being used only on the primary button component, and we shouldn't create lots of variables, because it's harder to maintain in the future.